src/components/GrantRequest/GrantRequestSectionCard.tsx

```diff
@@ -15,25 +26,22 @@ export const GrantRequestSectionCard = ({
   titleExtra,
   subtitleVariant = 'normal',
   error,
-  onClick,
-}: {
-  title: string | React.ReactNode
-  content: string | React.ReactNode | null
-  titleExtra?: string
-  subtitle: string | React.ReactNode
-  helper?: React.ReactNode
-  subtitleVariant?: 'normal' | 'uppercase'
-  error?: boolean
-  onClick?: () => void
-}) => {
+  href,
+}: Props) => {
   return (
-    <div
+    <a
```

Comment:
Can't we use the <Link /> component? I think it's the one from `decentraland-gatsby`.